hildon/hildon-entry.c

```diff
@@ -71,7 +71,11 @@ set_property                                    (GObject      *object,
 	/* If this is auto height, default to finger height. */
 	if (!(size & (HILDON_SIZE_FINGER_HEIGHT | HILDON_SIZE_THUMB_HEIGHT)))
 	  size |= HILDON_SIZE_FINGER_HEIGHT;
-	hildon_gtk_widget_set_theme_size (GTK_WIDGET (object), size);
+#ifdef MAEMO_GTK
+        hildon_gtk_widget_set_theme_size (GTK_WIDGET (object), size);
```

Comment:
Changing initial tab to spaces here seems redundant.
The commits history should reflect code changes, not your editor preferences.

Reply:
This was not tab -> spaces. For some reason I thought that this code was misindented, but it's not the case. I'll revert this one.

Reply:
I can only see tab to 8 spaces change in my text editor.

Reply:
The code around here had a messy mixture of tabs&spaces. I changed everything to spaces for consistency.